src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -1,70 +1,55 @@
+
+    toggle(event)
+    {
+        /** @type {HTMLElement[]} */
+        const contentTargets = this.contentTargets;
+
+        /** @type {HTMLElement[]} */
+        const headerTargets = this.headerTargets;
+
+        /** @type {HTMLElement[]} */
+        const arrowTargets = this.arrowTargets;
 
-    toggle(event) {
         const current = event.currentTarget;
-        const arrow = current.querySelector('[data-araise--table-bundle--accordion-target=arrow]');
-        const isOpen = current.dataset.ariaExpanded == 'true';
-        const nextSiblings = this.nextUntil(current, '[data-action="click->araise--table-bundle--accordion#toggle"]');
+        const header = this.closestTarget(current, 'header');
+        const arrow = this.closestTarget(current, 'arrow');
+        const content = header.nextElementSibling;
+
+        const isOpen = header.getAttribute('aria-expanded') === 'true';
 
-        if(isOpen) {
+        // Reset all accordions
+        headerTargets.forEach((header) => {
+            header.setAttribute('aria-expanded', 'false');
+        });
+        arrowTargets.forEach((arrow) => {
             arrow.classList.remove('rotate-90');
-            current.dataset.ariaExpanded = 'false';
+        });
+        contentTargets.forEach((content) => {
+            content.classList.add('hidden');
+        });
 
-            nextSiblings.forEach((sibling) => {
-                sibling.classList.add('hidden');
-            });
-        } else {
+        // Open the current accordion if it wasn't open before (else we toggle it)
+        if (!isOpen) {
             arrow.classList.add('rotate-90');
-            current.dataset.ariaExpanded = 'true';
-
-            nextSiblings.forEach((sibling) => {
-                sibling.classList.remove('hidden');
-            });
+            header.setAttribute('aria-expanded', 'true');
+            content.classList.remove('hidden');
         }
     }
 
-    /*!
-     * Get all following siblings of each element up to but not including the element matched by the selector
-     * (c) 2017 Chris Ferdinandi, MIT License, https://gomakethings.com
-     * @param  {Node}   elem     The element
-     * @param  {String} selector The selector to stop at
-     * @param  {String} filter   The selector to match siblings against [optional]
-     * @return {Array}           The siblings
-     */
-    nextUntil(elem, selector, filter) {
-
-        // Setup siblings array
-        var siblings = [];
-
-        // Get the next sibling element
-        elem = elem.nextElementSibling;
-
-        // As long as a sibling exists
-        while (elem) {
-
-            // If we've reached our match, bail
-            if (elem.matches(selector)) break;
-
-            // If filtering by a selector, check if the sibling matches
-            if (filter && !elem.matches(filter)) {
-                elem = elem.nextElementSibling;
-                continue;
+    closestTarget(element, target)
+    {
+        while (element) {
```

Comment:
Surely, there is a more elegant and stable solution than using a while selector. It's quite unclear to me what is happening in this function and while we «overwrite» an input parameter of the function directly. This smells like a hack. 

Why do we need this function and not use the native `Element.clostest(selector)` instead?